addons/account/models/account_journal_dashboard.py

```diff
@@ -413,6 +413,38 @@ def create_bank_statement(self):
         })
         return action
 
+    @api.multi
+    def create_customer_payment(self):
+        """return action to create a customer payment"""
+        action = self.env.ref('account.action_account_payments').read()[0]
+        action.update({
+            'views': [[False, 'form']],
+            'context': "{'default_payment_type': 'inbound', 'default_partner_type': 'customer', 'default_journal_id': " + str(self.id) + "}",
```

Comment:
you can write the context without "" :

'context': {'default_payment_type': 'inbound', 'default_partner_type': 'customer', 'default_journal_id': self.id)},